exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,7 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  return arr.length; // complete this statement
```

Comment:
Add if (arr.length === 0) {
    return true; // complete this statement
  } else {
    return false;
  } instead
Your code returns the length not the expected results of false or true